lib/Event/Ecommerce/IndexServiceEvents.php

```diff
@@ -33,4 +33,13 @@ final class IndexServiceEvents
      * @var string
      */
     const GENERAL_PREPROCESSING_ERROR = 'pimcore.ecommerce.indexservice.generalPreProcessingError';
+    
+       /**
+     * Fired when values in filter definition get fetched
+     *
+     * @Event("Symfony\Component\EventDispatcher\GenericEvent")
+     *
+     * @var string
+     */
+    const GET_VALUES_FOR_FILTER_FIELD_PRE_SEND_DATA = 'pimcore.ecommerce.indexservice.getValuesForFilterFieldPreSendData';
```

Comment:
Since this event isn't related to the index service, I don't think this is the right place for it. 
I think a dedicated class would make sense here, e.g. `\Pimcore\Event\Ecommerce\AdminEvents`.

Reply:
done